packages/scene-composer/src/components/panels/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,39 @@
+@use '@awsui/design-tokens/index.scss' as awsui;
+
+.color-picker .react-colorful {
+  width: auto
+}
+
+.color-picker .react-colorful__saturation {
+  border-radius: 5px;
+  border-bottom: none;
+}
+
+.color-picker .react-colorful__hue {
+  margin-top: 24px;
+  margin-bottom: 20px;
```

Comment:
Things that are assigning visual sizes such as margins, height, width, & padding should use `rem` or `em` in order to maintain responsive & consistent appearance between browsers & monitors. 

Things that control edges (borders, radius, etc...) are where use of `px` are appropriate. 

We have not standardized on whether we're going to be using `rem` or `em`, but would strongly encourage implementing either.

Reply:
Fixed

Reply:
Good point! Fixed

Reply:
Good point @mumanity